Core/Core/API/APIDiscussion.swift

```diff
@@ -32,10 +32,10 @@ public struct APIDiscussionTopic: Codable, Equatable {
 }
 
 public struct APIDiscussionAuthor: Codable, Equatable {
-    let id: String
-    let display_name: String
+    let id: String?
+    let display_name: String?
     let avatar_image_url: URL?
-    let html_url: URL
+    let html_url: URL?
```

Comment:
This and `APIRubric` were the primary culprits. For whatever reason the discussion assignments for this course do not have authors.